examples/ml-pipelines-node-classification.ipynb

```diff
@@ -0,0 +1,1540 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graphdatascience import GraphDataScience"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When using a local Neo4j setup, the default connection URI is `bolt://localhost:7687`. When using AuraDS, the connection URI is slightly different as it uses the `neo4j+s` protocol. In this case, the client should also include the `aura_ds=True` flag to enable AuraDS-recommended settings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 130,
+   "metadata": {
+    "id": "gJLQgHK_AghP"
+   },
+   "outputs": [],
+   "source": [
+    "# Replace with the actual connection URI and credentials\n",
+    "NEO4J_CONNECTION_URI = \"bolt://localhost:7687\"\n",
+    "NEO4J_USERNAME = \"neo4j\"\n",
+    "NEO4J_PASSWORD = \"test\"\n",
+    "\n",
+    "gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD))\n",
+    "\n",
+    "# On AuraDS:\n",
+    "#\n",
+    "# NEO4J_CONNECTION_URI = \"neo4j+s://xxxxxxxx.databases.neo4j.io\"\n",
+    "# NEO4J_USERNAME = \"neo4j\"\n",
+    "# NEO4J_PASSWORD = \"...\"\n",
+    "#\n",
+    "# gds = GraphDataScience(NEO4J_CONNECTION_URI, auth=(NEO4J_USERNAME, NEO4J_PASSWORD), aura_ds=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We also need to check that the version of the GDS library is 2.2.0 or newer, because we will use the concept of \"context\" introduced in [GDS 2.2.0](https://github.com/neo4j/graph-data-science/releases/tag/2.2.0)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 131,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert gds.version() >= \"2.2.0\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zOK08QpoWFKv"
+   },
+   "source": [
+    "Finally, we import `json` to help in writing the Cypher queries used to load the data, and `numpy` and `pandas` for further data processing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 132,
+   "metadata": {
+    "id": "-ikcvI-cSDoK"
+   },
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "oDE2LqTXAmy8"
+   },
+   "source": [
+    "## Load the Cora dataset\n",
+    "\n",
+    "First of all, we need to load the Cora dataset on Neo4j. The CSV files can be found at the following URIs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 133,
+   "metadata": {
+    "id": "tOZW-I2FBLm_"
+   },
+   "outputs": [],
+   "source": [
+    "CORA_CONTENT = (\n",
+    "    \"https://raw.githubusercontent.com/neo4j/graph-data-science/master/test-utils/src/main/resources/cora.content\"\n",
+    ")\n",
+    "CORA_CITES = (\n",
+    "    \"https://raw.githubusercontent.com/neo4j/graph-data-science/master/test-utils/src/main/resources/cora.cites\"\n",
```

Comment:
Yes, I'd much prefer to have them here along with the notebooks. I think there was a potential issue with licensing, but I assume it has been resolved (@Mats-SX knows better).

Reply:
Well, we already have Cora pickled as data frames here: https://github.com/neo4j/graph-data-science-client/tree/main/graphdatascience/resources. You could probably read those and take it from there?

Reply:
I can, although this would sort of hide the process behind reading and using an external source, and make a reader "commit" to our ready-to-use objects. (Also, the problem of writing the data back to Neo4j on Aura for the `write` example would remain.)

Reply:
I could add this as a comment maybe? Something on the lines of "For the Cora-specific example we already have the data ready in a suitable format, which you could load like this".

Reply:
Thanks @Mats-SX. I think the general agreement from today's discussion is that we keep the `LOAD CSV` section for the time being, for both the reasons stated. When the `write` action becomes available in Aura too, we can definitely revisit this.
Furthermore, @adamnsch noted that there will be a repository for datasets to be used later on in place of the files included in the GDS repo.

Reply:
https://trello.com/c/yeRoLpBX/7459-move-cora-dataset-sources-to-an-s3-bucket